src/plugins/github/README.md

```diff
@@ -0,0 +1,175 @@
+# GitHub Plugin
+
+This is the SourceCred GitHub plugin. It creates a cred graph based on data
+from [GitHub].
+
+[GitHub]: https://github.com/
+
+## Nodes
+
+The GitHub plugin creates the following kinds of nodes:
+
+### Repo
+
+A GitHub repository, e.g. [sourcecred/sourcecred]. It will be directly
+connected to all of the pulls and issues in the repository. Setting a weight on
+the repository will have no effect (i.e. they do not mint cred). This will
+change when we switch to [CredRank].
+
+[CredRank]: https://github.com/sourcecred/sourcecred/issues/1686
+
+[sourcecred/sourcecred]: https://github.com/sourcecred/sourcecred
+
+### Issue
+
+A GitHub issue, e.g. [sourcecred/sourcecred#40]. Issues are connected to their
+author(s), to entities that they reference, to their comments, and to the
+containing GitHub repository. If issues are given a positive weight, then each
+issue will mint cred. Note that this could be abused by spammers.
+
+[sourcecred/sourcecred#40]: https://github.com/sourcecred/sourcecred/issues/40
+
+### Pull Request
+
+A GitHub pull request, e.g. [sourcecred/sourcecred#35][pull]. Pulls are
+connected to their author(s), to entities they reference, to their comments,
+their reviews, to their containing repository, and, (if merged) to the commit
+that they merged as. If pulls are given a positive weight, then each pull will
+mint cred, regardless of whether or not it merged. Note that this could be
+abused by spammers. There's a proposal ([merge-minted cred]) to have pulls mint
+cred only when they merge.
+
+[pull]: https://github.com/sourcecred/sourcecred/pull/35
+[merge-minted cred]: https://github.com/sourcecred/sourcecred/issues/1682
+
+### Pull Request Review
+
+A review of a GitHub pull request, e.g. [this review]. Reviews are connected to
+their author(s), to entities they reference, to their comments, and to the pull
+they review. If a positive weight is set on the reviews, then every review will
+mint cred; this could be abused by spammers.
+
+[this review]: https://github.com/sourcecred/sourcecred/pull/91#pullrequestreview-105254836
+
+### Comment
+
+A comment on an issue, pull request, or pull request review. Comments are
+connected to their author(s), to entities they reference, and to their 'parent'
+(the containing issue, pull, or review). If a positive weight is set on
+comments, then every comment will mint cred. This can be absued by spammers,
+and also may encourage long bikeshedding discussions or flame wars.
```

Comment:
I think this is a fairly generalizing caution. Dropping comments to 0 in a community where the bulk of decisions are made by commenting on issues, could undervalue that discussion.

So the nuance I think is that setting it too high has a risk of leading to bikeshedding / flaming incentives.
Or similar to our Discourse vs GitHub weights might lead to a "rational actor" would stop doing things we need more. As comments could have a higher reward:effort ratio.

Reply:
I re-wrote to just have a "caveats" section up front that talks about the issues with activity minted cred.